kdump-switch/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Which licence should be used? This is using Kubernetes while others are using Google.

Reply:
Yes it should be Google license. Sorry.